backend/src/main/java/com/htecgroup/SkyNest/service/impl/UserServiceImpl.java

```diff
@@ -61,4 +77,21 @@ public UserDto findUserByEmail(String email) {
 
     return modelMapper.map(userEntity, UserDto.class);
   }
+
+  @Override
+  public Boolean confirmEmail(String token) {
+    boolean validateToken = jwtEmailVerificationUtils.validateJwtToken(token);
+    if (validateToken) {
+      String email = jwtEmailVerificationUtils.getEmailFromJwtEmailVerificationToken(token);
+      UserEntity userEntity =
+              userRepository
+                      .findUserByEmail(email)
+                      .orElseThrow(
+                              () -> new UsernameNotFoundException("could not find user with email: " + email));
+      userEntity.setEnabled(true);
+      userEntity.setVerified(true);
+      userRepository.save(userEntity);
+      return userEntity.getEnabled() && userEntity.getVerified();
```

Comment:
Due to the previous setters, isn't the return value always true?

Reply:
I always like to return something to be sure that it's working during testing and trying it out.. 
I can maybe return ResponseEntity of a String.